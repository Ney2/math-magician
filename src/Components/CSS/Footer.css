* {
  margin: 0;
  padding: 0;
}

.footer {
  flex-direction: column;
  position: fixed;
  bottom: 0;
  width: 100%;
}

.copyright {
  text-align: center;
  font-family: 'Roboto', sans-serif;
  font-size: 20px;
  color: blue;
  background-image: none;
}

@media (max-width: 768px) {
  body {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
  }

  .footer {
    text-align: center;
  }
}
