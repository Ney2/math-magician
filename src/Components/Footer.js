import React from 'react';
import './CSS/Footer.css';

function Footer() {
  return (
    <div className="footer">
      <div className="copyright">
        <h4>Designed by Kidist-Guta</h4>
        <p>Copyright@2022</p>
      </div>
    </div>
  );
}

export default Footer;
