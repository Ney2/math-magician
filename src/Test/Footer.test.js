import React from 'react';
import Renderer from 'react-test-renderer';
import Footer from '../Components/Footer';

it('renders correctly', () => {
    const tree = Renderer
        .create(<Footer/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});