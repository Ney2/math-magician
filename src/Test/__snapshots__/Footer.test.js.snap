// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="copyright"
  >
    <h4>
      Designed by Kidist-Guta
    </h4>
    <p>
      Copyright@2022
    </p>
  </div>
</div>
`;
